understanding/labels-or-instructions.html

```diff
@@ -58,182 +56,119 @@
                   
                </p>
-               <p>Further, this Success Criterion does not take into consideration whether or not alternative
-                  methods of
-                  providing an accessible name or description for form controls and inputs has been
-                  used - this aspect is
-                  covered separately by <a href="name-role-value">4.1.2: Name, Role and Value</a>. It is possible
-                  for controls and inputs to have an appropriate accessible name or description (e.g.
-                  using <code>aria-label="..."</code>)
-                  and therefore pass Success Criterion 4.1.2, but to still fail this Success Criterion
-                  (if the labels or instructions
-                  aren't presented to all users, not just those using assistive technologies).
-                  
+               <p>さらに、この達成基準では、フォームのコントロール及び入力欄にアクセシブルな名前 (accessible name) 又は説明を提供するための代替方法が使用されているかどうかは考慮されない。この観点は、<a href="name-role-value">4.1.2: 名前 (name)・役割 (role)・値 (value)</a> で個別に扱われる。コントロール及び入力欄が適切なアクセシブルな名前 (accessible name) 又は説明 (例えば、<code>aria-label="..."</code> を使用する) を持ち、よって達成基準 4.1.2 に合格することが可能であるが、この達成基準には失敗する (支援技術を使用しているユーザーだけでなく、すべてのユーザーにラベル又は指示が提示されていない) 場合がある。
                </p>
-               <p>This Success Criterion does not apply to links or other controls (such as an expand/collapse
-                  widget, or similar
-                  interactive components) that are not associated with data entry.
+               <p>この達成基準は、データ入力に関連しないリンク又はその他のコントロール (展開／折りたたみウィジェット、類似のインタラクティブなコンポーネントなど) には適用されない。
                   
                </p>
-               <p>While this Success Criterion requires that controls and inputs have labels or instructions,
-                  whether or
-                  not labels (if used) are sufficiently clear or descriptive is covered separately by
-                  <a href="headings-and-labels">2.4.6: Headings and Labels</a>.
+               <p>この達成基準では、コントロール及び入力にラベル又は説明があることを要求しているが、ラベル (使用されている場合) が十分に明確又は説明的であるかどうかについては、<a href="headings-and-labels">2.4.6: 見出し及びラベル</a>で別途扱われる。
                   
                </p>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
                   
-                  <li>Providing labels and instructions (including examples of expected
-                     data formats) helps all users - but particularly those with cognitive, language, and
-                     learning
-                     disabilities - to enter information correctly.
+                  <li>明確なラベル及び説明（予想されるデータフォーマットの例を含む）を提供すると、すべての利用者、特に認知障害、言語障害、学習障害のある利用者が、情報を正しく入力するのに役立つ。
```

Comment:
2.1 からの差分で「明確な (clear)」がなくなっているので、省いてよいかと思います。

```suggestion
                  <li>ラベル及び説明（予想されるデータフォーマットの例を含む）を提供すると、すべての利用者、特に認知障害、言語障害、学習障害のある利用者が、情報を正しく入力するのに役立つ。
```